string/view.mbt

```diff
@@ -447,3 +447,9 @@ pub fn StringView::iter(self : StringView) -> Iter[Char] {
     }
   })
 }
+
+///|
+/// Returns true if the view is empty.
+pub fn StringView::is_empty(self : StringView) -> Bool {
```

Comment:
This seems unused in this PR? I think it's better to put it in separated PR.

Reply:
The usage is removed in this commit https://github.com/moonbitlang/core/pull/1582/commits/d099cae1f6fa136db058b433106934bc504fc7a3. I will clean it up.